tests/test_cli.py

```diff
@@ -425,3 +426,12 @@ def test_parse_sort_first(arg, expected_first, expected_overrides):
     first, overrides = parse_sort_first(arg, {})
     assert first == expected_first
     assert overrides == expected_overrides
+
+
+def test_version():
+    """Test that `pretty-toml-sort --version` returns correct value."""
+    with open("pyproject.toml") as pyproject_toml_file:
```

Comment:
apparently needs encoding="UTF-8" to pass pylint - don't know why `pylint tests/test_cli.py` missed this on my own laptop 😕